.github/CODEOWNERS

```
@@ -652,6 +652,7 @@ peps/pep-0770.rst  @sethmlarson @brettcannon
 peps/pep-0777.rst  @warsaw
 # ...
 peps/pep-0789.rst  @njsmith
+peps/pep-0790.rst  @astrofrog @DEKHTIARJonathan @pradyunsg
```

Only https://github.com/python org members can be listed here, so let's add Pradyun. 

@pradyunsg Please could you confirm you're sponsoring this?

Also, let's use the next contiguous number from the main block, 771, and move this up the list, and update the actual PEP's filename.

```suggestion
peps/pep-0771.rst  @pradyunsg
```

---

Thanks! I'll commit the suggestion then will mark this as unresolved so that @pradyunsg sees the question